src/shared/components/testimonials/Testimonials.tsx

```diff
@@ -101,6 +100,12 @@ export default class Testimonials extends React.Component <{}, {}> {
                     <div className='testimonial-blockquote' key={testimonialIndex}>
                         <p>"{activeTestimonial.quote}"</p>
                         <cite>--{activeTestimonial.cite}</cite>
+                        <div id="testimonial-links">
```

Comment:
avoid using IDs accept at the highest level of a site.  i.e. page layout.   it's just a good practice to uses classes.  in this case there will probably not be another instance of this on the same page, but ids are a habit to avoid.